config.env

```diff
@@ -0,0 +1 @@
+export TEST_DB_PASSWORD=correct
```

Comment:
Done. 

I missed what the problem was for hanging tests and i do not know current linting situation so sorry if there are some issues.